fastn-core/tests/20-fastn-update-check/cmd.p1

```diff
@@ -0,0 +1,9 @@
+-- fbt:
+cmd: $FBT_CWD/../target/debug/fastn --test update --check
+exit-code: 1
```

Comment:
Return Exit code 7 to mean check failure. Every other error also returns exit code 1, so end user will have to look at error message instead of exit code to see what happened.